frontend/include/chpl/framework/error-classes-list.h

```diff
@@ -36,9 +36,13 @@ ERROR_CLASS(MemManagementNonClass, const uast::New*, const types::Type*)
 ERROR_CLASS(PrivateToPublicInclude, const uast::Include*, const uast::Module*)
 ERROR_CLASS(PrototypeInclude, const uast::Include*, const uast::Module*)
 ERROR_CLASS(MissingInclude, const uast::Include*, std::string)
-ERROR_CLASS(UseImportUnknown, const uast::VisibilityClause*,
+ERROR_CLASS(UseImportUnknownSym, const uast::VisibilityClause*,
             const resolution::Scope*, const resolution::VisibilityStmtKind,
             std::string)
+ERROR_CLASS(UnknownUseImport, const ID, const resolution::VisibilityStmtKind,
```

Comment:
Were you going for some kind of consistency between `UseImportUnknownSym` and `UnknownUseImport`? Perhaps the latter should be `UnknownUseImport`?

Note that error message names are part of the public API (c.f. `detailed.good` files)

Reply:
I assume you meant that the latter should be `UseImportUnknown`, since you typed the same name it currently is. If that's the case then I would be concerned that `UseImportUnknownSym` and `UseImportUnknown` are too similar of names; perhaps the second could be called `UseImportUnknownModule`.